dp_requirements/morpho_tagger.txt

```diff
@@ -0,0 +1 @@
+russian-tagsets
```

Comment:
It's better to use a specific version, as changes can break compatibility (or a package itself)